File: TP3-CPP/Kode Program/Peserta.cpp
#pragma once
#include <string>
#include <iostream>
using namespace std;

// Kelas Peserta untuk informasi dari seseorang yang mengikuti trip
class Peserta {
public:
    string no_ktp;
    string nama;
    int umur;
    string kontak;

    // Konstruktor untuk inisialisasi data peserta
    Peserta(string no_ktp, string nama, int umur, string kontak)
        : no_ktp(no_ktp), nama(nama), umur(umur), kontak(kontak) {}

    // Method untuk menampilkan informasi peserta ke layar
    void tampilkan() const {
        cout << "  - " << nama << "(KTP: " << no_ktp 
             << ", Umur: " << umur << ", Kontak: " << kontak << ")" << endl;
    }
};

File: TP3-Java/Kode Program/Peserta.java
public class Peserta {
    private String noKtp;
    private String nama;
    private int umur;
    private String kontak;

    /*onstruktor untuk menginisialisasi objek Peserta */
    public Peserta(String noKtp, String nama, int umur, String kontak) {
        this.noKtp = noKtp;
        this.nama = nama;
        this.umur = umur;
        this.kontak = kontak;
    }
    
    /* Menampilkan informasi peserta */
    public void tampilkan() {
        System.out.println("  - " + nama + " (KTP: " + noKtp + ", Umur: " + umur + ", Kontak: " + kontak + ")");
    }
}
